core/templates/dev/head/pages/base.html

```diff
@@ -72,6 +72,8 @@
       {% include 'pages/header_css_libs.html' %}
     {% endblock header_css %}
 
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/load_constants.js"></script>
```

Comment:
Sounds good, thanks for investigating. Worth adding a comment for devs IMO so they don't move things around in the future.